rasa/core/actions/forms.py

```diff
@@ -428,6 +441,12 @@ def _temporary_tracker(
             slots=domain.slots,
         )
 
+    async def user_rejected_manually(self, validation_events: List[Event]) -> bool:
```

Comment:
let's make it protected as nothing outside the class calls it 
```suggestion
    async def _user_rejected_manually(self, validation_events: List[Event]) -> bool:
```